app/assets/stylesheets/landing_page/sections/categories.scss

```diff
@@ -0,0 +1,356 @@
+@import "../mixins/all";
+
+$categories__desktop-height-7: 525px;
+$categories__desktop-height-6: 558px;
+$categories__desktop-height-5: 558px;
+$categories__desktop-height-4: 208px;
+$categories__desktop-height-3: 282px;
+
+$categories__tablet-height-7: 457px; // The height is actually more than this, but this is the
+                                     // number which is used to calculate the area based on the ratio
+$categories__tablet-height-6: 457px;
+$categories__tablet-height-5: 457px;
+$categories__tablet-height-4: 441px;
+$categories__tablet-height-3: 169px;
+
+$categories__mobile-height-big: 163px;
+$categories__mobile-height-small: 123px;
+
+$categories__desktop-gutter: 18px;
+$categories__tablet-gutter: 14px;
+$categories__mobile-gutter: 12px;
+
+.categories__section {
+  color: #525961;
+
+  width: 100%;
+  @include prefix-val(display, flex);
+  @include prefix-prop(align-items, center);
+
+  padding: $dimension-section-margin 0px;
+}
+
+.categories__content {
+  margin: 0 auto;
+  text-align: center;
+
+  // mobile
+  width: 100%;
+
+  @media #{$tablet} {
+    width: 100%;
```

Comment:
Unneeded if the mobile width still applies with this media query (and it isn't overriding width value set elsewhere).